app/services/users.js

```diff
@@ -3,6 +3,9 @@ const jwt = require('jwt-simple');
 const models = require('../models/index');
 const errors = require('../errors');
 const generatePassword = require('../helpers/users/generatePassword');
+const logger = require('../logger');
+
+const numberOfRecords = 10;
```

Comment:
This should not be hardcoded. Usually depending on the program, you can either simply configure it via environment variables, or have the front send it (If the front does not send it, it should take the default value from the env).